src/Main.cc

```diff
@@ -389,6 +401,7 @@ static int tclAppInit(int& argc,
                       const char* init_filename,
                       Tcl_Interp* interp)
 {
+  Tcl_SetAssocData(interp, "design", nullptr, the_design.get());
```

Comment:
Could you add a comment here why we're doing this, for future readers?

Reply:
done